_posts/0000-01-02-TokiThor.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Is this correct, I'm not sure 
![Ninja_idle_4](https://user-images.githubusercontent.com/76405258/102722901-4732a980-42d2-11eb-81fd-6374635df94e.png)